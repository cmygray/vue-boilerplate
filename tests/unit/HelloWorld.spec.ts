import { shallowMount, createLocalVue } from '@vue/test-utils';

import HelloWorld from '@/components/HelloWorld.vue';

const localVue = createLocalVue();

const PROPS = {
  msg: 'hello world'
};

describe('HelloWorld.vue', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallowMount(HelloWorld, {
      localVue,
      propsData: PROPS,
    });
  });

  it('renders as expected', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
