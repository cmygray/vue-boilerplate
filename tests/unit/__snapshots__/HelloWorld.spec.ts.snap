// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue renders as expected 1`] = `
<section class="hello-world">
  <h2>hello world</h2>
</section>
`;
